file_store/src/file_store.rs

```diff
@@ -198,6 +204,25 @@ impl FileStore {
             })
             .boxed()
     }
+
+    pub fn stream_files<A, B, F>(&self, file_type: F, after: A, before: B) -> Stream<FileData>
```

Comment:
I note the `listing` of files is handled internally here.  I think it might be better to allow the caller to list the files and then stream_files take this as an input.  It allows simple things like the caller to choose to log the number of files found, which is currently what the iot verifier does ( and is helpful to get an idea of whats going on ).  

Obviously you could log it here too but then that enforces the same behaviour for all consumers of the file store

Reply:
I added `stream_file` to make it up to caller how to loop over files